src/Microsoft.TestPlatform.CoreUtilities/Tracing/EqtTrace.cs

```diff
@@ -52,7 +52,7 @@ public static TraceLevel TraceLevel
 
 #endif
 
-#if NETSTANDARD || NET
+#if NETSTANDARD || NET || NETCOREAPP3_1
```

Comment:
is NET not defined for NETCOREAPP3_1?

Reply:
I have tested and no. `NET` doesn't work for `NETCOREAPP3_1`.